wincon/Makefile

```diff
@@ -55,10 +55,12 @@ endif
 # Only decision is:  are we doing a 64-bit compile (_w64 defined)?
 
 ifndef ON_WINDOWS
+    PREFIX  = i686-w64-mingw32-
```

Comment:
please use consistent spacing, so in this case a tab

Reply:
I copy-n-pasted the other lines, I fixed the modified lines.